src/Views/VPNPage.vala

```diff
@@ -43,34 +35,24 @@ public class Network.VPNPage : Network.Widgets.Page {
     }
 
     construct {
-        vpn_info_box = new Network.Widgets.VPNInfoBox ();
-        vpn_info_box.margin = 12;
-
-        popover = new Gtk.Popover (info_btn);
-        popover.position = Gtk.PositionType.BOTTOM;
-        popover.add (vpn_info_box);
-        popover.hide.connect (() => {
-            info_btn.active = false;
-        });
-
-        connected_frame = new Gtk.Frame (null);
-        connected_frame.get_style_context ().add_class (Gtk.STYLE_CLASS_VIEW);
-
-        top_revealer = new Gtk.Revealer ();
-        top_revealer.transition_type = Gtk.RevealerTransitionType.SLIDE_DOWN;
-        top_revealer.add (connected_frame);
+        var remove_vpn_toast = new Granite.Widgets.Toast ("");
+        remove_vpn_toast.set_default_action (_("Undo"));
+        remove_vpn_toast.title = _("VPN removed");
```

Comment:
Any reason why the title is set here instead of directly in the constructor?

Reply:
No. Just copied from somewhere. Will fix it :smile: